From 6b08dbbde393db80a32aa37b812892f7276d68f2 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Thu, 3 May 2018 15:26:34 +0300
Subject: [PATCH 083/448] sdk: dts: ppc/arm64: fix the number of QMan DCP SPs

The QMan has 16 sub-portals for each DCP on DPAA 1.x parts.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
---
 .../boot/dts/freescale/qoriq-qman-portals-sdk.dtsi |    2 +-
 .../boot/dts/fsl/qoriq-qman3-ceetm0-32-sdk.dtsi    |    2 +-
 .../boot/dts/fsl/qoriq-qman3-ceetm0-8-sdk.dtsi     |    2 +-
 .../boot/dts/fsl/qoriq-qman3-ceetm1-32-sdk.dtsi    |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/qoriq-qman-portals-sdk.dtsi b/arch/arm64/boot/dts/freescale/qoriq-qman-portals-sdk.dtsi
index 8dfb7db..d3abfaf 100644
--- a/arch/arm64/boot/dts/freescale/qoriq-qman-portals-sdk.dtsi
+++ b/arch/arm64/boot/dts/freescale/qoriq-qman-portals-sdk.dtsi
@@ -31,7 +31,7 @@
 	qman-ceetm@0 {
 		compatible = "fsl,qman-ceetm";
 		fsl,ceetm-lfqid-range = <0xf00000 0x1000>;
-		fsl,ceetm-sp-range = <0 12>;
+		fsl,ceetm-sp-range = <0 16>;
 		fsl,ceetm-lni-range = <0 8>;
 		fsl,ceetm-channel-range = <0 32>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-32-sdk.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-32-sdk.dtsi
index 02474c9..a6205b1 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-32-sdk.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-32-sdk.dtsi
@@ -35,7 +35,7 @@
 	qman-ceetm@0 {
 		compatible = "fsl,qman-ceetm";
 		fsl,ceetm-lfqid-range = <0xf00000 0x1000>;
-		fsl,ceetm-sp-range = <0 12>;
+		fsl,ceetm-sp-range = <0 16>;
 		fsl,ceetm-lni-range = <0 8>;
 		fsl,ceetm-channel-range = <0 32>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-8-sdk.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-8-sdk.dtsi
index 66cb8d9..779ab5e 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-8-sdk.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm0-8-sdk.dtsi
@@ -35,7 +35,7 @@
 	qman-ceetm@0 {
 		compatible = "fsl,qman-ceetm";
 		fsl,ceetm-lfqid-range = <0xf00000 0x1000>;
-		fsl,ceetm-sp-range = <0 12>;
+		fsl,ceetm-sp-range = <0 16>;
 		fsl,ceetm-lni-range = <0 8>;
 		fsl,ceetm-channel-range = <0 8>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm1-32-sdk.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm1-32-sdk.dtsi
index 91bb5f8..2c75f4d 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm1-32-sdk.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-qman3-ceetm1-32-sdk.dtsi
@@ -35,7 +35,7 @@
 	qman-ceetm@1 {
 		compatible = "fsl,qman-ceetm";
 		fsl,ceetm-lfqid-range = <0xf10000 0x1000>;
-		fsl,ceetm-sp-range = <0 12>;
+		fsl,ceetm-sp-range = <0 16>;
 		fsl,ceetm-lni-range = <0 8>;
 		fsl,ceetm-channel-range = <0 32>;
 	};
-- 
1.7.1

